daemon/images.go

```diff
@@ -96,7 +96,7 @@ func (r resources) IDs() (ids []flux.ResourceID) {
 }
 
 // getUnlockedAutomatedServices returns all the resources that are
-// both automated, and not locked.
+// both automated, and not locked or ignored.
 func (d *Daemon) getUnlockedAutomatedResources(ctx context.Context) (resources, error) {
```

Comment:
I would rename this to something more consistent/descriptive.

Perhaps `getAllowedAutoResources`, `filterAutomatedResources`, `getEffectivelyAutomated` ...?